gto-support-core/src/main/java/org/ccci/gto/android/common/util/AsyncTaskCompat.java

```diff
@@ -11,13 +11,37 @@
 import java.util.concurrent.TimeUnit;
 
 public class AsyncTaskCompat {
-    private static final Object LOCK_EXECUTOR = new Object();
-    private static Executor EXECUTOR;
+    private static final Compat COMPAT;
+    static {
+        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.HONEYCOMB) {
+            COMPAT = new FroyoCompat();
+        } else {
+            COMPAT = new HoneycombCompat();
+        }
+    }
+
+    public static void execute(@NonNull final Runnable task) {
+        COMPAT.execute(task);
+    }
 
     @NonNull
-    @TargetApi(Build.VERSION_CODES.GINGERBREAD)
-    private static Executor getExecutor() {
-        synchronized (LOCK_EXECUTOR) {
+    public static Executor SERIAL_EXECUTOR() {
```

Comment:
Not sure I like the all caps syntax for this method (vs something like `getExecutor()`). It implies wrapping a constant in a method call, which is true of Honeycomb, but not Froyo. Don't think I care enough to have you change it if you feel strongly about this syntax, though.